app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -408,6 +408,67 @@ def test_update_experiment_is_rollout(self):
         experiment = NimbusExperiment.objects.get(id=experiment_id)
         self.assertTrue(experiment.is_rollout)
 
+    def test_update_live_rollout(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            is_rollout=True,
+            population_percent=40,
+        )
+        experiment_id = experiment.id
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "populationPercent": "50",
+                    "changelogMessage": "test changelog message",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+
+        experiment = NimbusExperiment.objects.get(id=experiment_id)
+        self.assertTrue(experiment.is_rollout)
+        self.assertEqual(experiment.population_percent, 50.0)
+        self.assertEqual(experiment.publish_status, NimbusConstants.PublishStatus.IDLE)
+        self.assertEqual(experiment.status, NimbusConstants.Status.LIVE)
+        self.assertEqual(experiment.status_next, None)
+
+    def test_do_not_update_live_experiment(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            is_rollout=False,
+            population_percent=40,
+        )
+        experiment_id = experiment.id
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "populationPercent": "50",
+                    "changelogMessage": "test changelog message",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+
+        content = json.loads(response.content)
+        result = content["data"]["updateExperiment"]
+        self.assertEqual(result["message"].keys(), {"experiment"})
```

Comment:
Oh here's a good example:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/tests/api/v5/test_mutations.py#L740-L750

Reply:
You are correct, but in this case the name is literally "experiment" 😆 https://github.com/mozilla/experimenter/blob/b93b308e74d8806d05d12b22b98ae1db30f758a6/app/experimenter/experiments/api/v5/serializers.py#L668

Reply:
I could do some matching on the error message itself, but it's a cumbersome message and it ended up looking pretty gross and confusing

Reply:
Oh I see

```
{'message': {'experiment': ["Nimbus Experiment has Idle 'Live', only {'all': ['is_archived', 'publish_status', 'status_next', 'status', 'takeaways_summary', 'conclusion_recommendation'], 'experiments': [], 'rollouts': ['population_percent']} can be changed, not: {'changelog_message', 'population_percent'}"]}}
```

I thought it would raise on the field itself.  Yep looks good 👍